src/deploy/table_event_policy.sql

```diff
@@ -13,26 +14,31 @@ GRANT INSERT, UPDATE, DELETE ON TABLE maevsi.event TO maevsi_account;
 
 ALTER TABLE maevsi.event ENABLE ROW LEVEL SECURITY;
 
--- Only display events that are public and not full.
+-- Only display events that are public and not full and not organized by a blocked account
 -- Only display events that are organized by oneself.
--- Only display events to which oneself is invited.
+-- Only display events to which oneself is invited, but not by an invitation authored by a blocked account
 CREATE POLICY event_select ON maevsi.event FOR SELECT USING (
 -- Below copied to function `maevsi.event_invitee_count_maximum`.
-      (
-        visibility = 'public'
-        AND
-        (
-          invitee_count_maximum IS NULL
-          OR
-          invitee_count_maximum > (maevsi.invitee_count(id)) -- Using the function here is required as there would otherwise be infinite recursion.
-        )
-      )
-  OR  (
-    NULLIF(current_setting('jwt.claims.account_id', true), '')::UUID IS NOT NULL
+  (
+    visibility = 'public'
     AND
-    author_account_id = NULLIF(current_setting('jwt.claims.account_id', true), '')::UUID
+    (
+      invitee_count_maximum IS NULL
+      OR
+      invitee_count_maximum > (maevsi.invitee_count(id)) -- Using the function here is required as there would otherwise be infinite recursion.
+    )
```

Comment:
Just fyi: changing the indentation while also having changes to the code makes it hard to read the difference for reviewers as you can see in above. The code stays mostly the same here, but there are way more green and red lines than necessary for an addition. Therefore, it's better to change formatting in a separate PR. You can resolve this comment once understood.

Reply:
What are the general formatting rules?

Reply:
I aim to have our code in the same formatting as the schema file output eventually. For now, the formatting should generally be kept as is in the existing files. Reformatting can happen though of course, just in a separate PR please.